walkeeper/src/timeline.rs

```diff
@@ -748,53 +730,52 @@ mod test {
 
     fn load_from_control_file(
         conf: &SafeKeeperConf,
-        timeline_id: ZTimelineId,
+        zttid: &ZTenantTimelineId,
         create: CreateControlFile,
     ) -> Result<(FileStorage, SafeKeeperState)> {
-        fs::create_dir_all(&conf.timeline_dir(&timeline_id))
-            .expect("failed to create timeline dir");
+        fs::create_dir_all(&conf.timeline_dir(zttid)).expect("failed to create timeline dir");
         Ok((
-            FileStorage::new(timeline_id, conf),
-            FileStorage::load_control_file_conf(conf, timeline_id, create)?,
+            FileStorage::new(zttid, conf),
+            FileStorage::load_control_file_conf(conf, zttid, create)?,
         ))
     }
 
     #[test]
     fn test_read_write_safekeeper_state() {
         let conf = stub_conf();
-        let timeline_id = ZTimelineId::generate();
+        let zttid = ZTenantTimelineId::new(ZTenantId::generate(), ZTimelineId::generate());
```

Comment:
May be cleaner to have ZTenantTimelineId::generate

Reply:
ok